backend.js

```diff
@@ -2,32 +2,39 @@ const express = require("express");
 const cors = require("cors");
 
 const taskServices = require("./models/taskServices");
-const Task = require("./models/taskSchema");
 
 const app = express();
 const port = 5000;
 
 app.use(cors());
 app.use(express.json());
 
-app.get("/", (req, res) => { /* all the tasks */
-  const result = taskServices.getTasks();
-  res.send(result);
+app.get("/", async (req, res) => {
+  /* testing endpoints */
+  res.send("TODOLIST");
 });
 
-app.get("/tasks", async (req, res) => {
-  const name = req.query["name"];
+app.get("/task", async (req, res) => {
+  /* dump all tasks */
+  const result = await taskServices.getTasks();
+  res.send({ task_list: result });
+});
+
+app.get("/task/:id", async (req, res) => {
+  /* get by task id */
+  const id = req.params.id;
   try {
-    const result = await taskServices.getTasks(name);
+    const result = await taskServices.findTaskById(id);
     res.send({ task_list: result });
   } catch (error) {
     console.log(error);
     res.status(500).send("An error ocurred in the server.");
```

Comment:
change error return value, it looks like 500 is being used for this particular get and in post